FEATURETEST.md

```diff
@@ -0,0 +1,21 @@
+#### Takeaway-challenge Feature-test
+
```

Comment:
This document is a great way to explain how to run and operate your code, I really like how you've broke down each function!